packages/block-editor/src/components/block-tools/insertion-point.js

```diff
@@ -294,6 +300,8 @@ function InsertionPointPopover( {
 			// Forces a remount of the popover when its position changes
 			// This makes sure the popover doesn't animate from its previous position.
 			key={ nextClientId + '--' + rootClientId }
+			onFocusOutside={ maybeHideInserterPoint }
+			onMouseLeave={ maybeHideInserterPoint }
 		>
 			<motion.div
```

Comment:
I've started looking at this and I'm not familiar with this code, but I was wondering if you have search for a solution from [motion](https://www.framer.com/docs/gestures/###onhoverend) maybe something like `onHoverEnd={ maybeHideInserterPoint }` in this motion.div?

Reply:
> I was wondering if you have search for a solution from motion maybe something like onHoverEnd={ maybeHideInserterPoint } in this motion.div?

Thanks for looking!

I tried it out and I don't see any difference in behaviour, so I don't see any implementation issues with it.

Was there a specific reason why we should prefer motion's eventhandler?

Reply:
> Was there a specific reason why we should prefer motion's eventhandler?

Oh, I did find one advantage, aside from the fact that `Popover` doesn't specifically define an `onMouseLeave` event: we can perform an equality check on the even target and the current ref to make sure the event is occurring on the motion div. 

I've pushed that change since I couldn't detect any noticeable divergence in behaviour.